xbmc/pvr/PVRDatabase.cpp

```diff
@@ -757,10 +757,11 @@ bool CPVRDatabase::Persist(CPVRChannel &channel, bool bCommit)
     return bReturn;
   }
 
-  std::string strQuery;
   CSingleLock lock(m_critSection);
 
-  if (channel.ChannelID() <= 0)
+  std::string strQuery = PrepareSQL("iUniqueId = %u AND iClientId = %u", channel.UniqueID(), channel.ClientID());
```

Comment:
Maybe add a comment here so someone doesn't refactor this later in search for performance gains?

Reply:
Good point, will add a comment.